frontend/projects/upgrade/src/app/features/dashboard/dashboard-root/dashboard-root.component.ts

```diff
@@ -18,11 +18,11 @@ export class DashboardRootComponent implements OnInit {
       text: 'global.experiment.title',
       iconType: 'assignment',
     },
-    // {
-    //   path: ['/featureFlags'],
-    //   text: 'feature-flags.title.text',
-    //   iconType: 'toggle_on'
-    // },
+    {
```

Comment:
can we actually use this PR to just set a "Feature Flag" feature flag boolean in the ENV? Then we'd be able to wrap the nav item / route in flag condition and start working on Feature-Flag in the 'dark', whatever we end up doing with existing functionality.

Reply:
I have no idea what to name it though. FEATURE_FLAG_FEATURE_FEATURE_FLAG 😵‍💫